src/scarab.js

```diff
@@ -124,6 +149,7 @@ var MapzenScarab = (function () {
     linkEl.href = linkHref
     linkEl.target = '_blank'
     linkEl.className = 'mz-bug-' + id + '-link'
+    linkEl.classList.add('mz-bug-icons')
```

Comment:
We can't use `classList` in very low-level libraries because it's not supported below IE10. We have to add it directly on the string value for the `.className` property (above line).